static/js/components/PersonalTab.js

```diff
@@ -25,19 +26,27 @@ class PersonalTab extends React.Component {
   updateGender(gender) {
     const { dispatch, profile: { profileCopy } } = this.props;
```

Comment:
What if I just call it `edit` instead of `unsavedProfile`? It feels weird to call it `unsavedProfile` if it contains `{ errors, profile }`, and it can be accessed by `edit.profile` or `edit.errors` which seems nice to read